build/index.js

```diff
@@ -402,7 +402,8 @@ var CRAWLER_USER_ID = core.getInput('crawler-user-id');
 var CRAWLER_API_KEY = core.getInput('crawler-api-key');
 var CRAWLER_API_BASE_URL = core.getInput('crawler-api-base-url');
 var GITHUB_TOKEN = core.getInput('github-token');
-var CRAWLER_NAME = core.getInput('crawler-name').replace(/\//g, '-');
+var CRAWLER_NAME = core.getInput('crawler-name').replace(/[ /]/g, '-');
```

Comment:
we don't need to remove space neither slash for crawler name, it currently makes the name hardly readable.
the indexname spec should be covered by next replace